parsing/parser.mly

```diff
@@ -2351,6 +2378,7 @@ simple_expr:
       { array_get ~loc:$sloc $1 $4 }
   | simple_expr DOT LPAREN seq_expr error
       { unclosed "(" $loc($3) ")" $loc($5) }
+  (* Immutable array indexing is a regular operator *)
```

Comment:
I don't understand the import of this comment.

Reply:
This should've gotten clarified in the rebase, there are a couple new comments about immutable array indexing now